source/features/show-whitespaces.css

```diff
@@ -0,0 +1,23 @@
+.pl-ws {
```

Comment:
If these are our own classes, they should start with `rgh` to make it clear.

If GitHub also uses these, ignore this comment